lib/compiler/src/beam_types.erl

```diff
@@ -382,7 +382,12 @@ subtract(#t_union{tuple_set=#t_tuple{}=Tuple}=A, #t_tuple{}=B) ->
 subtract(#t_union{other=Other}=A, B) ->
     shrink_union(A#t_union{other=subtract(Other, B)});
 
-subtract(T, _) -> T.
+subtract(A, B) ->
+    %% There's nothing left if A is strictly more specific than B.
```

Comment:
I think that the word *strictly* should be omitted. There will be nothing left if the types are equally specific.

Reply:
Good point, fixed locally.